tests/models/fast/test_image_processing_fast.py

```diff
@@ -0,0 +1,191 @@
+            numpify=numpify,
+            torchify=torchify,
+        )
+
+
+@require_torch
+@require_vision
+class FastImageProcessingTest(ImageProcessingTestMixin, unittest.TestCase):
+    image_processing_class = FastImageProcessor if is_vision_available() else None
+
+    def setUp(self):
+        self.image_processor_tester = FastImageProcessingTester(self)
+        self.image_processor_list = []
+
+    @property
+    def image_processor_dict(self):
+        return self.image_processor_tester.prepare_image_processor_dict()
+
+    def test_image_processor_properties(self):
+        image_processing = self.image_processing_class(**self.image_processor_dict)
+        self.assertTrue(hasattr(image_processing, "do_resize"))
+        self.assertTrue(hasattr(image_processing, "size"))
+        self.assertTrue(hasattr(image_processing, "do_center_crop"))
+        self.assertTrue(hasattr(image_processing, "center_crop"))
+        self.assertTrue(hasattr(image_processing, "do_normalize"))
+        self.assertTrue(hasattr(image_processing, "image_mean"))
+        self.assertTrue(hasattr(image_processing, "image_std"))
+
+    def test_image_processor_from_dict_with_kwargs(self):
+        image_processor = self.image_processing_class.from_dict(self.image_processor_dict)
+        self.assertEqual(image_processor.size, {"height": 20, "width": 20})
+        self.assertEqual(image_processor.crop_size, {"height": 18, "width": 18})
+
+        image_processor = self.image_processing_class.from_dict(
+            self.image_processor_dict, size=42, crop_size=84, reduce_labels=True
+        )
+        self.assertEqual(image_processor.size, {"shortest_edge": 42})
+        self.assertEqual(image_processor.crop_size, {"height": 84, "width": 84})
+
+    # @slow
+    def test_post_process_text_detection(self):
+        model = FastForSceneTextRecognition.from_pretrained("jadechoghari/fast_tiny_ckpt")
+
+        image_processor = FastImageProcessor.from_pretrained("jadechoghari/fast_tiny_ckpt")
+
+        def prepare_image():
+            image_url = "https://huggingface.co/datasets/Raghavan/fast_model_samples/resolve/main/img657.jpg"
+            raw_image = Image.open(requests.get(image_url, stream=True).raw).convert("RGB")
+            return raw_image
+
+        image = prepare_image()
+        inputs = image_processor(image, return_tensors="pt")
+        with torch.no_grad():
+            output = model(pixel_values=inputs["pixel_values"])
+        target_sizes = [(image.height, image.width)]
+        threshold = 0.88
+        final_out = image_processor.post_process_text_detection(
+            output, target_sizes, threshold, bounding_box_type="rect"
+        )
+        assert final_out[0]["boxes"][0] == [
+            (148, 151),
+            (157, 53),
+            (357, 72),
+            (347, 170),
+        ]  # (151, 151), (160, 56), (355, 74), (346, 169)
+        assert round(float(final_out[0]["scores"][0]), 5) == 0.92573  # 0.91862
```

Comment:
please not use `assert` as it -> self.assertTrue/self.assertListEqual/etc  

or torch.testing.assert_close(...) for tensors